Makefile

```diff
@@ -245,14 +249,14 @@ endif
 SERVICE_ACCOUNTS=--extra-service-accounts dynatrace-dynakube-oneagent
 SERVICE_ACCOUNTS+=--extra-service-accounts dynatrace-dynakube-oneagent-unprivileged
 SERVICE_ACCOUNTS+=--extra-service-accounts dynatrace-kubernetes-monitoring
-SERVICE_ACCOUNTS+=--extra-service-accounts dynatrace-activegate
 
 # Generate bundle manifests and metadata, then validate generated files.
-.PHONY: bundle
-bundle: export OLM=true
-bundle: export OUT=olm
-bundle: manifests kustomize
-	operator-sdk generate kustomize manifests -q
+.PHONY: bundle bundle-olm
+# to avoid calling make bundle with OUT=olm
```

Comment:
how does it avoid running bundle with OUT=olm? doesn't it set it?
`make bundle-olm OLM=true OUT=olm`

or do you mean "calling make bundle with**out** OUT=olm"?

Reply:
I mean calling make bundle without OUT=olm, to avoid breaking your pipline and also to avoid that overhead